### ISSUE TITLE
Could not get unknown property 'rnMinorVersion'

### ISSUE THREAD

**Issue by Wentris71** [software-mansion/react-native-reanimated#3247]
### Expected behavior
to build app successfully.

## Package versions

| name                                 | version |
| ------------------------------------ | ------- |
| react-native                         |  0.69.0-rc.0       |
| react-native-reanimated              |  2.8.0       |
| NodeJS                               | v14.17.6       |
| Xcode                                | Version 12.5 (12E262)        |
| Java                                 |         |
| Gradle                               |  distributions/gradle-7.3.3-all.zip - gradle:7.1.1     |

## Affected platforms

- [ x] Android
- [ ] iOS
- [ ] Web

**Comment by dilarakirn** [comment]
@Wentris71 Hi, is there any solution for this problem ?

**Comment by denisk20** [comment]
Same when running with react native 0.69.0-rc.4. I have fixed this by changing `$rnMinorVersion` to `rnMinorVersionCopy` in `detectAAR` def in `node_modules/react-native-reanimated/android/build.gradle`. This issue is valid.

**Comment by Wentris71** [comment]
@dilarakirn hey, I simply downgraded RN from 0.69.0-rc.0 to 0.68.2

**Comment by nzcodarnoc** [comment]
I'm having this issue on "react-native": "0.69.0" and "react-native-reanimated": "2.8.0",

**Comment by feraswfares** [comment]
same issu here +1

"react-native": "0.69.0",
"react-native-reanimated": "2.8.0",


```
Could not get unknown property 'rnMinorVersion' for project ':react-native-reanimated' of type org.gradle.api.Project
```

**Comment by shensven** [comment]
+1

**Comment by piaskowyk** [comment]
I have released version 2.9.1 with a fix for that issue.
